src/core/lib/transition-manager.js

```diff
@@ -8,6 +8,9 @@ import {
 
 const noop = () => {};
 
+const VIEWPORT_TRANSITION_PROPS =
+  ['longitude', 'latitude', 'zoom', 'bearing', 'pitch'];
```

Comment:
`position` is not included? Is that intentional?

Reply:
Same reason as I replied to comment in `viewport-animation-first-person-map/app.js`.